scss/src/main/resources/sass/styleguide/index.scss

```diff
@@ -24,7 +24,7 @@
 }
 
 #body-wrapper.style-guide {
-  width: 1000px;
+  // width: 1000px;
```

Comment:
Have you taken a look at this https://wiki.openmrs.org/display/docs/Java+Conventions?